peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -129,10 +147,68 @@ <h5>{% translate 'Related documents' %}</h5>
       </div>
     </div>
 
-    {{ citation_links|json_script:"citation-links" }}
+        {{ citation_links|json_script:"citation-links" }}
 
-    {% block content %}
-      {% include 'peachjam/_document_content.html' with document=document %}
-    {% endblock %}
+        {% block content %}
+          {% include 'peachjam/_document_content.html' with document=document %}
+        {% endblock %}
+      </div>
+      <div class="tab-pane fade" id="history-tab" role="tabpanel" aria-labelledby="history-tab">
+        <div class="card">
```

Comment:
Move all of this into a separate include, it'll be much easier to reason about.

Reply:
refactored with nick